src/main/java/org/mockito/internal/configuration/injection/ConstructorInjection.java

```diff
@@ -26,13 +28,8 @@
  * and try to resolve mocks by type.
  * </p>
  *
- * <blockquote>
- * TODO on missing mock type, shall it abandon or create "noname" mocks.
- * TODO and what if the arg type is not mockable.
- * </blockquote>
- *
  * <p>
- * For now the algorithm tries to create anonymous mocks if an argument type is missing.
+ * The algorithm tries to create anonymous mocks if an argument type is missing.
  * If not possible the algorithm abandon resolution.
```

Comment:
Very minor typo, it should be algorithm abandons* resolution.
```suggestion
 * If not possible the algorithm abandons resolution.
```

Reply:
Nice catch! Thank you!